### PR TITLE
first commit

### PR DESCRIPTION
**Pull request by AlexForJava**

### CODE REVIEW

File: src/main/java/com/gmil/chernii/oleksii/dao/UserDao.java [hunk 1]
```diff
@@ -0,0 +1,20 @@
...
+
+import java.util.List;
+
+/**
+ * Created by Space on 23.04.2019.
+ */
+public interface UserDao {
+    void insert(User user);
+
+    User getById(Long id);
```
Comment (Taniann) [inline src/main/java/com/gmil/chernii/oleksii/dao/UserDao.java]: better return here `Optional<User>`

File: src/main/java/com/gmil/chernii/oleksii/dao/UserDaoImpl.java [hunk 2]
```diff
@@ -0,0 +1,87 @@
...
+        if (INSTANCE == null) {
+            synchronized (UserDaoImpl.class) {
+                if (INSTANCE == null) {
+                    INSTANCE = new UserDaoImpl();
+                }
+            }
+        }
+        return INSTANCE;
+    }
+
```
Comment (Taniann) [inline src/main/java/com/gmil/chernii/oleksii/dao/UserDaoImpl.java]: remove redundant blank line

**Review by Taniann — CHANGES_REQUESTED** [review]
please resolve comments

File: src/main/java/com/gmil/chernii/oleksii/HibernateUtil.java [hunk 3]
```diff
@@ -0,0 +1,25 @@
+package com.gmil.chernii.oleksii;
+
+import com.gmil.chernii.oleksii.entity.User;
+import org.hibernate.SessionFactory;
+import org.hibernate.boot.registry.StandardServiceRegistryBuilder;
+import org.hibernate.cfg.Configuration;
+import org.hibernate.service.ServiceRegistry;
+
+/**
+ * Created by Space on 23.04.2019.
```
Comment (asulyma) [inline src/main/java/com/gmil/chernii/oleksii/HibernateUtil.java]: drop it please

**Review by Taniann — COMMENTED** [review]
good

**Pull request merged**